### PR TITLE
Fixing a failing test

### PR DESCRIPTION
**Pull request by mdbenjam**
A recent change removed some CSS classes. A Queue test relied on that class to click a button, but the test also had randomness so it wouldn't always fail.

### CODE REVIEW

File: spec/feature/queue/checkout_flows_spec.rb [hunk 1]
```diff
@@ -482,27 +482,19 @@ def select_issue_level_options(opts)
         expect(page).to have_content("Evaluate Decision")
 
         click_on "Continue"
-        expect(page).to have_content("Choose one")
-        sleep 2
```
Comment (shanear) [inline spec/feature/queue/checkout_flows_spec.rb]: ooh, sleeps, no bueno

File: .circleci/config.yml [hunk 2]
```diff
@@ -136,6 +136,10 @@ jobs:
           name: Store capybara screenshots
           path: ~/project/tmp/capybara
 
+      - store_artifacts:
+          name: Store rails test log
+          path: ~/project/logs/test.log
```
Comment (mdbenjam) [inline .circleci/config.yml]: Not sure, but I *think* this is the right path.

File: spec/feature/queue/checkout_flows_spec.rb [hunk 3]
```diff
@@ -482,27 +482,19 @@ def select_issue_level_options(opts)
...
+        sleep 1
```
Comment (mdbenjam) [inline spec/feature/queue/checkout_flows_spec.rb]: Couldn't figure out how to get rid of this one 🤔 but I brought it down to one second for now.

File: spec/feature/queue/checkout_flows_spec.rb [hunk 4]
```diff
@@ -482,27 +482,19 @@ def select_issue_level_options(opts)
...
+        find("label", text: "Easy").click
+        find("label", text: "1 - Does not meet expectations").click
 
-        [case_complexity_opts, case_quality_opts].each { |l| l.sample(1).first.click }
         # areas of improvement
-        page.find_all(".question-label").sample(2).each(&:double_click)
+        find("#issues_are_not_addressed", visible: false).sibling("label").click
 
-        dummy_note = generate_words 200
+        dummy_note = generate_words 5
```
Comment (mdbenjam) [inline spec/feature/queue/checkout_flows_spec.rb]: 200 words seems excessive. Bringing it down to 5

File: spec/feature/queue/checkout_flows_spec.rb [hunk 5]
```diff
@@ -482,27 +482,19 @@ def select_issue_level_options(opts)
...
-        case_quality_opts = page.find_all(:xpath, "//fieldset[@class='#{radio_group_cls}'][2]//label")
+        expect(page).to have_content("Choose one")
 
-        expect(case_quality_opts.first.text).to eq(
-          "5 - #{Constants::JUDGE_CASE_REVIEW_OPTIONS['QUALITY']['outstanding']}"
-        )
-        expect(case_quality_opts.last.text).to eq(
-          "1 - #{Constants::JUDGE_CASE_REVIEW_OPTIONS['QUALITY']['does_not_meet_expectations']}"
-        )
+        find("label", text: "Easy").click
```
Comment (mdbenjam) [inline spec/feature/queue/checkout_flows_spec.rb]: Making these clicks deterministic.

File: .circleci/config.yml (hunk 2 above)
Comment (mdbenjam) [reply .circleci/config.yml]: Verified.

**Pull request merged**